src/vstd_ext/seq_lib.rs

```diff
@@ -243,6 +243,37 @@ pub proof fn true_pred_on_all_element_equal_to_pred_on_all_index<A>(s: Seq<A>, p
     }
 }
 
+#[verifier(external_body)]
+pub proof fn push_to_set_eq_to_set_insert<A>(s: Seq<A>, elt: A)
+    ensures s.push(elt).to_set() == s.to_set().insert(elt);
+
+#[verifier(external_body)]
```

Comment:
Both of these `verifier(external_body)` annotations mean these aren't proved yet.